packages/babel-core/src/config/full.ts

```diff
@@ -68,10 +68,9 @@ export default gensync<(inputOpts: unknown) => ResolvedConfig | null>(
       throw new Error("Assertion failure - plugins and presets exist");
     }
 
-    const pluginContext: Context.FullPlugin = {
+    const presetContext: Context.FullPreset = {
```

Comment:
Will `presetContext.assumptions` be `undefined`?

Reply:
Yes, but it doesn't matter because the preset API doesn't have a method to read assumptions anyway.